packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.stories.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, {
+  StorybookEmailArgs,
+  commonArgs,
+} from '../../storybook-email';
+
+export default {
+  title: 'Emails/postVerify',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
```

Comment:
Since this is duplicated in every story we may as well export it from `storybook-email.ts` and just import it into each story file, no?